src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -88,12 +92,18 @@ def rsyncResults(stage, duffyKey) {
  *
  * variables
  *  stage - current stage running
+ *  checkRsyncDir - boolean to determine if our URL path will be the rsync dir or not. Defaults to true
  */
-def checkLastImage(stage) {
+def checkLastImage(stage, checkRsyncDir=true) {
     echo "Currently in stage: ${stage} in checkLastImage"
+    if (checkRsyncDir==true){
```

Comment:
When would it not use RSYNC_DIR

Reply:
There were calls to an existing checkLastImage() method in PipelineUtils.groovy which had a path of:
`${HTTP_BASE}/${RSYNC_DIR}/${branch}/images/latest-atomic.qcow2`
however, in Jenkinsfile, there was also a definition of checkLastImage() which had a path of:
`{HTTP_BASE}/${branch}/images/latest-atomic.qcow2`
So it appears that the answer to the question of when would it not use RSYNC_DIR is, "when called from the jenkinsfile"?

If the expectation is that it should always reference the path with the RSYNC_DIR, I'll update the method.

Reply:
I don't think it uses anything else AFAIK

Reply:
I'll update it so that it automatically uses the path with the RSYNC_DIR

Reply:
cool